### DISCUSSION TITLE
Is there a way to have sign_update just print the generated signature?

### DISCUSSION THREAD

**Discussion by tonyarnold** [sparkle-project/Sparkle#2263] [Q&A]
Currently, running the `sign_update` tool prints out the following:

```
sparkle:edSignature="…" length="…"
```

This is great for manually generating an appcast XML directly (assuming you're just inserting the output as raw text), but it isn't great if you need those values discretely for other purposes (such as submitting to an API that generates your appcast for you. 

I checked the documentation and couldn't find any mention of accessing these values discretely. Is this not possible, or am I missing something?

**Comment by zorgiepoo** [comment]
Not currently possible, but this could be added if someone wants to make a pull request. It should be an easy addition. `generate_keys` is similar to `sign_update` here and has a `-p` option to only print the key. `sign_update` could add such an option too for the signature.

**Comment by zorgiepoo** [comment]
I made a simple enhancement change for it here https://github.com/sparkle-project/Sparkle/pull/2268
